deployments/data100/config/common.yaml

```diff
@@ -44,6 +44,18 @@ jupyterhub:
         # this role will be assigned to...
         groups:
           - course::1531798::group::Admins
+      # Econ 148, Spring 2024, DH-225
+      course-staff-1532866:
+        description: Enable course staff to view and access servers.
+        # this role provides permissions to...
+        scopes:
+          - admin-ui
+          - list:users!group=course::1532866
+          - admin:servers!group=course::1532866
+          - access:servers!group=course::1532866
+        # this role will be assigned to...
+        groups:
+          - course::1532866::group::Admins
```

Comment:
@balajialg Do you know if the Econ 148 course staff created this group in bCourses? If not, you would need to use the automatically created `::enrollment_type::ta` style groups. If they did create the group then don't mind me. :)

Reply:
![image](https://github.com/berkeley-dsep-infra/datahub/assets/2306166/1d9b34b8-4b6f-4852-b117-829a84b8a947)
@ryanlovett They seem to have a group named admin!

Reply:
I created the group! Although it's named `admin` not `Admins`